client/interfaces.go

```diff
@@ -160,6 +160,7 @@ type ContainerServer interface {
 	UpdateStoragePoolVolume(pool string, volType string, name string, volume api.StorageVolumePut, ETag string) (err error)
 	DeleteStoragePoolVolume(pool string, volType string, name string) (err error)
 	RenameStoragePoolVolume(pool string, volType string, name string, volume api.StorageVolumePost) (err error)
+	CopyStoragePoolVolume(source ContainerServer, pool string, volume api.StorageVolumesPost, args *StoragePoolVolumeCopyArgs) (err error)
```

Comment:
So unfortunately we kinda messed up storage in the API design by making the majority of calls synchronous when they should probably have been made async...

I don't feel too good about introducing a CopyStoragePoolVolume that's synchronous though so I think we need to make that particular create call return a background operation. 

This means that CopyStoragePoolVolume should instead return "(op *RemoteOperation, err error)" in line with what's done for container copies. This will become even more important as we add support for doing this remotely.